oxidation/client/package.json

```diff
@@ -11,7 +11,7 @@
         "test:unit": "cypress run --component",
         "test:e2e": "cypress run --e2e",
         "web:open": "vite --open",
-        "web:release": "vite build && run lint",
+        "web:release": "vite build",
```

Comment:
lint should stay (damn json format that prevented us from adding a comment !!!! ><'')
The issue is `vite build` doesn't do all the check on the typescript code (it's not called "vite" for nothing), so we also have to run the lint after to ensure the code is correct (and hence it compilation has produced a valid output)

Reply:
see https://vitejs.dev/guide/features.html#typescript

> Transpile Only
> Note that Vite only performs transpilation on .ts files and does NOT perform type checking. It assumes type checking is taken care of by your IDE and build process.

Reply:
done